README.md

```diff
@@ -0,0 +1,41 @@
+# onebusaway-travel-behavior-analysis
+Python utilities to process and analyze travel behavior data collected by OneBusAway and exported as csv file by oba-firebase-export.
+
+## matchAndMerge utility
+This utility matches ground truth data activities in excel format with activities data generated by oba-firebase-export in csv format. The algorithm will match each activity in the ground truth dataset with the nearest activity in the oba-firebase-export generated file. 
+
+## Build 
```

Comment:
I believe this and subsequent headings should be down one heading level `###`.

Reply:
I will include this section after re-attaching the requirements file to the project.